src/backend/core/admin.py

```diff
@@ -0,0 +1,89 @@
+"""Admin classes and registrations for People's core app."""
+from django.contrib import admin
+from django.contrib.auth import admin as auth_admin
+from django.utils.translation import gettext_lazy as _
+
+from . import models
+
+
+class IdentityInline(admin.TabularInline):
+    """Inline admin class for user identities."""
+
+    fields = (
+        "sub",
+        "email",
+        "is_main",
+        "created_at",
+        "updated_at",
+    )
+    model = models.Identity
+    extra = 0
+    readonly_fields = ("email", "created_at", "sub", "updated_at")
```

Comment:
<img src="https://github.com/numerique-gouv/people/assets/1427165/90769187-c711-419c-a117-3681af5195d7" width="200"/>

Good catch! In fact we should forbid creating identities via the admin as they are automatically created on successful logins.

I think we should keep the possibility for admins to modify the main identity. This can be useful for support and people logged in as superuers should know what they are doing?

Reply:
Great, thank you ! 

And I agree, for admins changing the main identify, it will definitely come handy for support.